tests/searching/task.yaml

```diff
@@ -0,0 +1,38 @@
+summary: Check snap search
+execute: |
+  echo "List all snaps"
+  expected="(?s)Name +Version +Developer +Notes +Summary *\n\
+  .*?\n\
+  canonical-pc +.*? *\n\
+  .*?\
+  canonical-pc-linux +.*? *\n\
+  .*?\
+  go-example-webserver +.*? *\n\
+  .*?\
+  hello-world +.*? *\n\
+  .*?\
+  ubuntu-clock-app +.*? *\n\
+  .*?\
+  ubuntu-core +.*? *\n\
```

Comment:
This is a test that will most likely be broken very soon, as we won't want to list everything.

For now, can we please keep just hello-world on the list, which is something we depend on for other tests anyway, so we reduce the changes of this test being broken on other unrelated changes?